Practica real con tabla/ViewController.swift

```diff
@@ -43,49 +43,46 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
 
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
 
+        // use large numbers to avoid collisions with system-defined views
+        let messageViewTag = 2
+        let ballonViewTag = 4
+        let labelTag = 8
```

Comment:
⬆️  
You should always try to use larger numbers to avoid collision with system defined views.
**There's a view with tag `0` in the `contentView` of `UITableViewCell`.**

Also, I prefer setting the numbers that will be used multiple times as variables or constants. 
This way, we can avoid careless mistakes like typo (since your app will not compile).